airbyte-cdk/java/airbyte-cdk/typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/DefaultTyperDeduperTest.java

```diff
@@ -94,12 +149,15 @@ void emptyDestination() throws Exception {
     initialStates.forEach(initialState -> when(initialState.isFinalTablePresent()).thenReturn(false));
 
     typerDeduper.prepareSchemasAndRawTables();
-    verify(destinationHandler).execute(separately("CREATE SCHEMA overwrite_ns", "CREATE SCHEMA append_ns", "CREATE SCHEMA dedup_ns"));
+    verify(destinationHandler).execute(separately("CREATE SCHEMA airbyte_internal", "CREATE SCHEMA overwrite_ns", "CREATE SCHEMA append_ns", "CREATE SCHEMA dedup_ns"));
```

Comment:
previously the tests didn't call `create schema airbyte_internal` because the StreamIds had null rawNamespace. We're now setting those to `airbyte_internal`, so these tests need to be updated.